UnityProject/Assets/Scripts/UI/Systems/ServerInfoUILobby.cs

```diff
@@ -29,7 +29,8 @@ public class ServerInfoUILobby : MonoBehaviour, IInitialise
 
         public static string serverDesc;
 
-        public static string serverDiscordID;
+        private static string serverDiscordID;
+        public string ServerDiscordID => serverDiscordID;
```

Comment:
```suggestion
        public string ServerDiscordID { get; private set;}
```

That way you don't need the private field.